src/components/markdown/markdown.module.css

```
@@ -34,10 +34,23 @@
+  border: 0;
   color: var(--button-desktop-text-color);
+  display: inline;
+  font: inherit;
+  line-height: inherit;
+  margin: 0;
+  padding: 0;
   text-transform: capitalize;
 }
 
+.embedButton:focus-visible {
+  outline: 1px dotted currentColor;
```

Stylelint (17.12.0)

[error] Expected "currentColor" to be "currentcolor" (value-keyword-case)

(value-keyword-case)